crates/query/fixtures/solana/queries/instruction_account/query.json

```diff
@@ -0,0 +1,38 @@
+{
+  "type": "solana",
+  "fromBlock": 0,
+  "fields": {
+    "block": {
+      "parentHash": true,
+      "timestamp": true,
+      "hash": true,
+      "height": true,
+      "number": true,
+      "parentNumber": true
+    },
+    "instruction": {
+      "programId": true,
+      "accounts": true,
+      "data": true,
+      "error": true,
+      "hasDroppedLogMessages": true,
+      "transactionIndex": true,
+      "instructionAddress": true
+    },
+    "transaction": {
+      "signatures": true,
+      "feePayer": true,
+      "err": true,
+      "transactionIndex": true
+    }
+  },
+  "instructions": [
+    {
+      "account": [
```

Comment:
should this query return instructions with BOTH accounts included? or it should behave as `or`
@eldargab

Reply:
It should be an `or`

Reply:
`or` would be more natural because other filters work in `or` manner. but then it will not be possible to filter instructions that contain multiple accounts... but yeah it's probably better to deal with api that has consistent behaviour...

Reply:
couldn't come up with a better naming than "account"...